tests/test-sampling.cpp

```diff
@@ -7,7 +7,9 @@
 
 #include <cmath>
 #include <numeric>
+
 #include <cassert>
+
```

Comment:
```suggestion
#include <cassert>
```

Reply:
Remove the extra lines please

Reply:
done in [b06dca6](https://github.com/ggerganov/llama.cpp/pull/2690/commits/b06dca678b251c94e05aedffa8aecc69c2bea388)